ViAn/GUI/videowidget.cpp

```diff
@@ -425,6 +435,24 @@ void VideoWidget::analysis_btn_clicked() {
     }
 }
 
+void VideoWidget::analysis_play_btn_clicked() {
```

Comment:
since the analysis play button is checkable it has the toggled() signal. this signal will emit a bool with the toggle state. might be worth checking out